pkg/model/model.go

```diff
@@ -48,11 +56,19 @@ func (r *Ref) GetTargetsByKind(kinds ...string) []LocatedResourceStatus {
 	return targets
 }
 
+// Match returns true if this Hostname is equal to the short or long v of a dns name
+func (h *HostName) Match(name string) bool {
+	if h.Name == name || fmt.Sprint(h.Name, ".", h.Namespace, ".svc.cluster.local") == name {
+		return true
+	}
+	return false
```

Comment:
Maybe something like this for better readability?

```suggestion
	equalsShortName := h.Name == name
	equalsFullDNSName := fmt.Sprint(h.Name, ".", h.Namespace, ".svc.cluster.local") == name
	return equalsShortName || equalsFullDNSName
```